pallets/oracle/src/lib.rs

```diff
@@ -269,15 +269,15 @@ impl<T: Config> Pallet<T> {
 		let converted = match (amount.currency(), currency_id) {
 			(x, y) if x == y => amount.amount(),
 			(_, _) => {
-				// first convert to wrapped, then convert wrapped to the desired currency
-				let base = Self::collateral_to_wrapped(amount.amount(), amount.currency())?; // maybe this does not work?
-				Self::wrapped_to_collateral(base, currency_id)?
```

Comment:
maybe we should switch the function calls here to align it more to what you would expect from the comment. ie
```
let base = Self::currency_to_usd(amount.amount(), amount.currency())?;

Self::usd_to_currency(base, currency_id)?
```

Reply:
I think it should not make a difference if we change the order